Trunk Based Development

gsampaio-rh opened the pull request.

mcanoy commented on Jenkinsfile:
```diff
@@ -56,32 +56,14 @@ pipeline{
             }
             when {
-              expression { GIT_BRANCH ==~ /(.*master|.*develop|.*feature.*)/ }
+              expression { GIT_BRANCH ==~ /(.*master)/ }
```
when blocks not needed here. Assume only running against master.

gsampaio-rh commented:
Ready to review. I added two conditional steps:

release tag
master branch
In order to push it to quay I added a secret entry in the openshift applier.
*The secret was created manually on S11

Quay repo is just a "fake". Once that we have it the correct Quay repo, we can change it.

gsampaio-rh commented:
@oybed All set to review :)
